src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4361,12 +4361,6 @@ private static bool IsValidAccessToScopedProperty(IExternalControl lhsControl, I
                    (currentControl.IsComponentControl ||
```

Comment:
the `isBehaviorOnly` argument to this function is now unused, consider removing it?

Reply:
Good idea, done